Havit.Blazor.Components.Web.Bootstrap.Documentation/Pages/Components/Concepts/SettingsAndDefaults_Documentation.razor

```diff
@@ -0,0 +1,69 @@
+﻿@page "/concepts/SettingsAndDefaults"
+@page "/concepts/Settings"
+@page "/concepts/Defaults"
+
+<h1>Settings and defaults</h1>
+<DocAlert Type="DocAlertType.Info">Although most components support the presented functionalities, some components were constructed without settings or defaults because it wouldn't add sufficient value.</DocAlert>
+
+<h2 id="Settings">Settings</h2>
+<p>Component settings are a way of specifying component parameters. They let you bundle parameters together. That way, component settings can be easily sent across the application and applied in multiple places without the need to send the values individually. This comes in handy for instance when storing user preferences.</p>
+
+<SectionTitle Id="SettingsDemo">Demo</SectionTitle>
+<p>Applying component settings is as easy as instantiating a settings object for the target component and assigning it to the <code>Settings</code> parameter as shown below.</p>
+<Demo Type="typeof(SettingsAndDefaults_Demo_Settings)" />
+
+<SectionTitle Id="Overriding" />
+<p>When a component decides what value to adhere to it heeds this flow: <strong>parameter > settings > defaults > exception</strong> meaning that when the parameter property is <code>null</code>, then the value is taken from settings, if that is also <code>null</code>, then the value is taken from application-wide defaults. In case all of the above are <code>null</code>, an exception is thrown since this is an unsupported scenario.</p>
```

Comment:
No need to mention the "exception". It can occur only in case of invalid defaults.